noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -108,15 +108,10 @@ impl ContextInterface for PrivateContext {
 
 impl PrivateContext {
     pub fn new(inputs: PrivateContextInputs, args_hash: Field) -> PrivateContext {
-        let side_effect_counter = inputs.start_side_effect_counter;
-        let mut min_revertible_side_effect_counter = 0;
-        if is_empty(inputs.call_context.msg_sender) {
-            min_revertible_side_effect_counter = side_effect_counter;
```

Comment:
`min_revertible_side_effect_counter` is only propagated to the public inputs in init kernel circuit. And it's on the entrypoint to decide if it has a value or not. It's not necessary to set it to be the same as the call's start counter if it's the first call (empty msg_sender).